src/actions/articleActions/ArticleActions.js

```diff
@@ -0,0 +1,94 @@
+import {
+  CREATE_ARTICLE,
+  GET_ARTICLES,
+  GET_SINGLE_ARTICLE,
+  EDIT_ARTICLE,
+  DELETE_ARTICLE
+} from "../actionTypes";
+
+import { HRK_URL, token } from "./index";
+
+const createArticles = article => dispatch => {
+  return fetch(`${HRK_URL}/articles/`, {
+    method: "POST",
+    headers: {
+      "content-type": "application/json",
+      authorization: `Bearer ${token}`
+    },
+    body: JSON.stringify({ article: article })
+  })
+    .then(res => res.json())
+    .then(res => {
+      dispatch({ type: CREATE_ARTICLE, payload: res.article });
+    })
+    .catch(error => error);
+};
+
+const getArticles = () => dispatch => {
+  return fetch(`${HRK_URL}/articles`, {
```

Comment:
this line is going to fail on Heroku. please change it to `${HRK_URL}articles` and add the trailing slash in your `.env` file.
Your current statement for the URL will evaluate to `https://ah-backend-athena-staging.herokuapp.com/admin/api//articles` - note the `//` after api
 Also, please use meaningful variable names (even if they are long). `BACKEND_URL` or `API_URL` would be better options to `HRK_URL`

Reply:
Thanks @bibangamba for this feedback. I am running the application an it works well with hweroku. We need to meet an checkout this together I think